data/custom/dizzyandy-sprites.toml

```diff
@@ -452,3 +452,13 @@ text_metaobj = {sprite = "text_metaobj", color = [0, 2], tiling = "none"}
 
 "text_it'll" = {sprite = "text_it'll", color = [5, 3], tiling = "none"}
 
+text_flora = {sprite = "text_flora", color = [5, 1], tiling = "none"}
+
+text_sway = {sprite = "text_sway", color = [1, 1], tiling = "none"}
+
+text_sustain = {sprite = "text_sustain", color = [2, 4], tiling = "none"}
+
+text_sustain = {sprite = "text_true", color = [0, 3], tiling = "none"}
```

Comment:
this doesn't seem intentional, duplicate keys don't work

Reply:
You can just pull it and fix it before you restart the bot though, right?

Reply:
i would. rather it be fixed on github.

Reply:
I believe you misunderstood. You can pull this and fix it in a separate commit before the changes are applied to the bot itself.